### DISCUSSION TITLE
[Feature Request] to use Staubli robots with offsets

### DISCUSSION THREAD

**Discussion by eduardsik** [visose/Robots#143] [Q&A] [answered]
Description of the problem: Some types of staubli robots display an offset, which causes the vertical axes to not coincide. Currently, the plugin does not support robots with offset.
I would like an extension of the plugin for these types of robots.
These are the following three robots:
Staubli TX (2) 40, 60 (L), 90 (L)

<img width="572" alt="Screenshot 2022-02-26 at 23 18 52" src="https://user-images.githubusercontent.com/4760893/155900228-c6e1acab-01cc-401f-b5e5-8934e664ed7f.png">
<img width="582" alt="Screenshot 2022-02-26 at 23 19 12" src="https://user-images.githubusercontent.com/4760893/155900231-ef61e1d1-f9bc-46a4-9c63-2821dcfd94cf.png">
<img width="574" alt="Screenshot 2022-02-26 at 23 19 35" src="https://user-images.githubusercontent.com/4760893/155900233-0fd21de9-caa4-497d-9241-3e2be347cda7.png">

**Comment by visose** [comment, marked as answer]
Robots like the TX60 will now work on the upcoming release, see https://github.com/visose/Robots/discussions/103#discussioncomment-3391046
